themes/openideal_theme/templates/node/node.html.twig

```diff
@@ -0,0 +1,96 @@
+{#
```

Comment:
What is the purpose of this template? Did we change anything here?

Reply:
I have removed the link to the node page, as Zohar wrote: "without linking to the full node page."